### DISCUSSION TITLE
increase font size of entity

### DISCUSSION THREAD

**Discussion by Iqbal198** [punxaphil/custom-sonos-card#482] [Q&A] [answered]
how can i increase font size of entity?
![Screenshot 2024-09-27 195428](https://github.com/user-attachments/assets/f11af901-3a4f-48c7-82d7-2cd3086cac29)

**Comment by punxaphil** [comment]
There are lots of card mod examples in the readme

**Comment by Iqbal198** [comment]
Thanks for replying.

I have card mod but I’m not too good with yaml.

I can use the examples but don’t know how to do what I want.

Can you give me a a simple increase font size code to copy and paste?


Sent from my iPhone

On 27 Sep 2024, at 20:06, Punxa Phil ***@***.***> wrote:

﻿

There are lots of card mod examples in the readme

**Comment by punxaphil** [comment]
card_mod:
  style:
    sonos-player$ sonos-player-header$: |      .entity {

        font-size: 12px !important;
      }



fre 27 sep. 2024 kl. 21:10 skrev Iqbal198 ***@***.***>:

> Thanks for replying.
>
> I have card mod but I’m not too good with yaml.
>
> I can use the examples but don’t know how to do what I want.
>
> Can you give me a a simple increase font size code to copy and paste?
>
>
> Sent from my iPhone
>

**Comment by punxaphil** [comment, marked as answer]
```
card_mod:
  style:
    sonos-player$ sonos-player-header$: |
      .entity {
        font-size: 12px !important;
      }
```

**Comment by Iqbal198** [comment]
Thanks allot. That worked :)
